src/compiler/codegen_c_json_parser.c

```diff
@@ -1371,7 +1371,7 @@ static int gen_struct_parser(fb_output_t *out, fb_compound_type_t *ct)
     println(out, "");
     /* TODO: */
 #if 0
-    println(out, "static inline int %s_parse_json_as_root(flatcc_builder_t *B, flatcc_json_parser_t *ctx, const void *buf, size_t bufsiz, int flags, const char *fid)", snt.text);
+    println(out, "static inline int %s_parse_json_as_root(flatcc_builder_t *B, flatcc_json_parser_t *ctx, const char *buf, size_t bufsiz, int flags, const char *fid)", snt.text);
```

Comment:
I think you were describing this declaration but seems it doesn't isn't used at the moment.

Reply:
yes - see other comment